import/PopupStackView.qml

```diff
@@ -0,0 +1,195 @@
+/*******************************************************************************
+ *  Copyright 2012-2018 Esri
```

Comment:
2020

Reply:
Sorry I've been copying these docs verbatim and didn't give them a proper once-over.